examples/plot_whole_brain_alignment.py

```diff
@@ -0,0 +1,261 @@
+    roi_map=clustering_img,
+    hemi="left",
+    view="lateral",
+    title="Ward parcellation on the left hemisphere",
+)
+
+###############################################################################
+# Fitting the alignment operator
+# ------------------------------
+# We use the computed labels to fit a local alignment procedure using
+# Optimal Transport (OT).
+
+from fmralign import PairwiseAlignment
+
+data_source_train = masker.transform(surf_source_train)
+data_target_train = masker.transform(surf_target_train)
+
+alignment_estimator = PairwiseAlignment(
+    method="ot",
+    labels=labels,
+)
+
+alignment_estimator.fit(data_source_train, data_target_train)
+
+#################################################################################
+# Simulating an activation patch
+# ------------------------------
+# We simulate some data on the source subject by generating a spherical
+# patch of activation on the temporal lobe.
+
+import numpy as np
+
+center_vertex_idx = 2150
+
+simulated = np.zeros((1, data_source_train.shape[1]))
+vertices = surf_source_train.mesh.parts["left"].coordinates
+distances = np.linalg.norm(vertices - vertices[center_vertex_idx], axis=1)
+simulated[0, : simulated.shape[1] // 2] = distances < 15
+
+plotting.plot_surf_roi(
+    fsaverage_meshes["inflated"],
+    masker.inverse_transform(simulated),
+    title="Simulated activation on the source subject",
+)
+
+#################################################################################
+# Transporting the activation patch
+# ---------------------------------
+# We use the fitted alignment operator to transport the simulated patch of
+# activation to the target subject. We then visualize the voxels that have been
+# affected by the transport operation and overlay the contours of the
+# ROIs. We observe that the parcellation
+# introduces some undesirable boundary effects. In particular, we observe that
+# some voxels are completely unaffected on the inferior temporal gyrus.
+
+import matplotlib.pyplot as plt
+
+affected_voxels = masker.inverse_transform(
+    alignment_estimator.transform(simulated) != 0
+)
+
+fig, ax = plt.subplots(1, 1, subplot_kw={"projection": "3d"})
+
+plotting.plot_surf_roi(
+    fsaverage_meshes["inflated"],
+    affected_voxels,
+    title="Transported Voxels",
+    axes=ax,
+    colorbar=False,
+)
+plotting.plot_surf_contours(
+    fsaverage_meshes["inflated"],
+    masker.inverse_transform(labels),
+    axes=ax,
+)
+#################################################################################
+# Soft constraints using the Laplacian embedding
+# ----------------------------------------------
+# To alleviate the boundary effects introduced by parcellation, we can
+# introduce soft constraints based on the geometry of the cortical surface.
+# Here, we compute the eigenfunctions of the Laplace-Beltrami operator on the
+# cortical mesh using the :func:`~fmralign.embeddings.whole_brain.get_laplacian_embedding`
```

Comment:
Since this is a new function, we need to add the rst stub for this to cross-link